py/htm/advanced/algorithms/thalamus.py

```diff
@@ -0,0 +1,361 @@
+# ----------------------------------------------------------------------
+# HTM Community Edition of NuPIC
+# Copyright (C) 2019, Numenta, Inc. 
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.    If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+
+An implementation of thalamic control and routing as proposed in the Cosyne
+submission:
+
+    A dendritic mechanism for dynamic routing and control in the thalamus Carmen Varela & Subutai Ahmad
```

Comment:
nit, this article could be found, but additional link would be fine

Reply:
I really wanted a copy of that poster but could not find it. The closest was a photo of the poster in one of Subutai's tweets. :-(.
If you can find it that would be great.

Reply:
hmm, atleast here is an abstract: https://numenta.com/company/events/cosyne-2019/ 
@rhyolight can we get the poster as a PDF, or ideally published on Numenta.org/papers, please? 
If none of that works, atlease we can link to the tweets (what a time :) )

Reply:
I'll update the Readme when there is something substantial. The photo in the tweet unfortunately was unreadable. At the time, I had requested a copy in the forum from Subutai but he did not respond. Maybe he didn't see it.